### ISSUE TITLE
ParticleTransformer training for decay mode classification is unstable

### ISSUE THREAD

**Issue by jpata** [HEP-KBFI/ml-tau-en-reg#22]
As seen here: https://github.com/HEP-KBFI/ml-tau-en-reg/pull/21#issuecomment-2129914582:
The validation loss of ParticleTransformer for `training_type=dm_multiclass` fluctuates like crazy.
Possible options:
- check what is different between predictions from epoch to epoch that introduces large fluctuations
- merge some rare decay modes so the classes are better represented
- increase dropout
- lower lr even further

**Comment by jpata** [comment]
Fixed in #23.
